pycm/pycm_curve.py

```diff
@@ -96,9 +97,60 @@ def area(self, method="trapezoidal"):
                 raise pycmCurveError(AREA_METHOD_ERROR)
         return self.auc
 
-    def plot(self):
-        """Plot method."""
-        pass
+    def plot(
+            self,
+            classes=None,
+            area=False,
+            method="trapezoidal",
+            colors=None,
+            marker=None,
+            linewidth=1):
+        """Plot the given curve.
+
+        :param classes: ordered labels of classes
+        :type classes: list
+        :param area: area flag
+        :type area: bool
+        :param method: numerical integral technique (trapezoidal or midpoint)
+        :type method: str
+        :param colors: color for each class in plot
+        :type colors: list
+        :param marker: plot marker
+        :type marker: str
+        :param linewidth: plot line width
+        :type linewidth: float
+        :return: plot axes
+        """
+        try:
+            from matplotlib import pyplot as plt
+        except (ModuleNotFoundError, ImportError):
```

Comment:
Please change this line to a general exception (like `Exception`).
It's not the best method, but it leads to some errors in **Python 3.5** tests.
We should drop **Python 3.5** support in the next release